backend/src/test/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceControllerTest.java

```diff
@@ -167,6 +172,148 @@ void verifyLinkV2ReturnsTestResultInfo() throws Exception {
     assertLastAuditEntry(HttpStatus.OK, ResourceLinks.VERIFY_LINK_V2, requestId);
   }
 
+  @Test
+  void verifyLinkV2ReturnsTestResultInfo_multiplex() throws Exception {
+    TestUserIdentities.withStandardUser(
+        () -> {
+          testEvent =
+              _dataFactory.createMultiplexTestEvent(
+                  person,
+                  facility,
+                  TestResult.POSITIVE,
+                  TestResult.NEGATIVE,
+                  TestResult.NEGATIVE,
+                  false);
+          patientLink = _dataFactory.createPatientLink(testEvent.getTestOrder());
+        });
+
+    // GIVEN
+    String dob = person.getBirthDate().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    String requestBody =
+        "{\"patientLinkId\":\""
+            + patientLink.getInternalId()
+            + "\",\"dateOfBirth\":\""
+            + dob
+            + "\"}";
+
+    // WHEN
+    MockHttpServletRequestBuilder builder =
+        post(ResourceLinks.VERIFY_LINK_V2)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .accept(MediaType.APPLICATION_JSON)
+            .characterEncoding("UTF-8")
+            .content(requestBody);
+
+    // THEN
+    String requestId =
+        mockMvc
+            .perform(builder)
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.testEventId", is(testEvent.getInternalId().toString())))
+            .andExpect(jsonPath("$.results", Matchers.hasSize(3)))
+            .andExpect(
+                jsonPath("$.results[?(@.disease.name == \"COVID-19\" && @.result == \"POSITIVE\")]")
```

Comment:
The results come back as a Set, so unlike some of these other `jsonPaths` we can't assert on the exact order the results will come back in. This regex allows us to assert that somewhere in the set, we expect a disease with the name COVID-19 to exist, and that at that `result.result`, it should be POSITIVE.